src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -1513,10 +1508,7 @@ namespace winrt::Microsoft::Terminal::Control::implementation
 
         _focused = false;
 
-        if (_uiaEngine.get())
-        {
-            THROW_IF_FAILED(_uiaEngine->Disable());
-        }

```

Comment:
make sure to update the method description since you're removing the _uiaEngine here